src/ci/resources/papi_application.inc.conf.ctmpl

```diff
@@ -63,3 +66,13 @@ services {
 	}
   }
 }
+
+filesystems.drs.global.config.martha.url = "https://us-central1-broad-dsde-dev.cloudfunctions.net/martha_v2"
+
+drs {
+  localization {
+    docker-image = "broadinstitute/cromwell-dos:34-d8acfe3"
+    #docker-image = "salonishah/dos:34-d8acfe3-with-println"
```

Comment:
Remove

Reply:
I forgot to clean this up 🤦‍♀ Will do it now. Thanks!